.github/workflows/Release.yml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2023 Mozilla Foundation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+name: Release Action CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump_version:
+        description: "Bump up to the next version and create a tag"
+        default: false
+        type: boolean
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the repository
+      - name: Checkout repository
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      # Setup Node.js
+      - name: Setup Node.js
+        uses: actions/setup-node@v5
+        with:
+          node-version: 20
+
+      # Install dependencies
+      - name: Install dependencies
+        run: npm ci
+
+      # Build setup.ts and show_stats.ts
+      - name: Build TypeScript files
+        run: |
+          npx @vercel/ncc build src/setup.ts -o dist/setup --minify
+          npx @vercel/ncc build src/show_stats.ts -o dist/show_stats --minify
+
+      # Determine version
+      - name: Determine next version
+        id: get_version
+        run: |
+          git fetch --tags
+          latest_tag=$(git describe --tags --abbrev=0 || echo "v0.0.0")
```

Comment:
Using '|| echo' to suppress commit errors could hide real issues, no ?

Reply:
also, why v0.0.0 as default ?

Reply:
Anyone who uses a fork and doesn't fetch tags would hit this case fwiw.